WizardEngine/Entity.cpp

```diff
@@ -1,10 +1,24 @@
 #include "Entity.h"
 #include "Component.h"
 
-Entity::Entity(Mesh * a_pMesh, Material* a_pMaterial, DirectX::XMFLOAT4X4 a_mWorld, DirectX::XMFLOAT3 a_vPos, DirectX::XMFLOAT3 a_vRotation, DirectX::XMFLOAT3 a_vScale) : m_pMesh(a_pMesh), m_mWorld(a_mWorld),
-m_vPos(a_vPos), m_vRotation(a_vRotation), m_vScale(a_vScale), m_pMaterial(a_pMaterial)
+using namespace DirectX;
```

Comment:
Using namespace to cut down on repetitive bulk. I think we should do this throughout the codebase